package.json

```diff
@@ -38,7 +38,7 @@
     "sprite": "scripts/make-icon-sprite.sh",
     "jest": "yarn test:jest",
     "test": "yarn test:jest",
-    "test:jest": "env BABEL_ENV=transpilation jest --verbose --coverage",
+    "test:jest": "env BABEL_ENV=test jest --verbose --coverage",
```

Comment:
No need for this. Since jest sets NODE_ENV to `test`, babel will have its env as `test`.

https://babeljs.io/docs/en/options#envname

> envName
> 
> Type: string
> Default: process.env.BABEL_ENV || process.env.NODE_ENV || "development"
> Placement: Only allowed in Babel's programmatic options

https://jest-bot.github.io/jest/docs/babel.html

> Note: If you are using a more complicated Babel configuration, using Babel's env option, keep in mind that Jest will automatically define NODE_ENV as test. It will not use development section like Babel does by default when no NODE_ENV is set.

Reply:
Thanks!